deploy/src/loadEnv.js

```diff
@@ -0,0 +1,68 @@
+const { isAddress, toBN } = require('web3').utils
+const path = require('path')
+const envalid = require('envalid')
+const { ZERO_ADDRESS } = require('./constants')
+
+const dotEnvPath = path.join(__dirname, '..', '.env')
+
+// Validations and constants
+const validBridgeModes = ['NATIVE_TO_ERC', 'ERC_TO_ERC', 'ERC_TO_NATIVE']
+const bigNumValidator = envalid.makeValidator(x => toBN(x))
+const validateAddress = address => {
+  if (isAddress(address)) {
+    return address
+  }
+
+  throw new Error(`Invalid address: ${address}`)
+}
+const addressValidator = envalid.makeValidator(validateAddress)
+const addressesValidator = envalid.makeValidator(addresses => {
+  addresses.split(' ').forEach(validateAddress)
+  return addresses
+})
+
+const env = envalid.cleanEnv(
+  process.env,
+  {
+    BRIDGE_MODE: envalid.str({ choices: validBridgeModes }),
+    DEPLOYMENT_ACCOUNT_PRIVATE_KEY: envalid.str(),
+    DEPLOYMENT_GAS_LIMIT: bigNumValidator(),
+    HOME_DEPLOYMENT_GAS_PRICE: bigNumValidator(),
+    FOREIGN_DEPLOYMENT_GAS_PRICE: bigNumValidator(),
+    GET_RECEIPT_INTERVAL_IN_MILLISECONDS: bigNumValidator(),
+    BRIDGEABLE_TOKEN_NAME: envalid.str(),
```

Comment:
The `.env` file without the variables `BRIDGEABLE_TOKEN_NAME`, `BRIDGEABLE_TOKEN_SYMBOL` and `BRIDGEABLE_TOKEN_DECIMALS` will cause inability to run deployment where as the bridge mode `erc-to-native` does not require them at all.

```
================================
 Missing environment variables:
    BRIDGEABLE_TOKEN_NAME: undefined
    BRIDGEABLE_TOKEN_SYMBOL: undefined
    BRIDGEABLE_TOKEN_DECIMALS: undefined

 Exiting with error code 1
================================
```

Reply:
I changed the way the env variables are validated to make more explicit which ones depend on the mode. Now it's more explicit. There is some duplication when a variable is needed in two modes but not in the other, but I think that's OK for now.